### PR TITLE
asar: add page

### PR DESCRIPTION
**Pull request by idleberg**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by mfrw** [comment]
Merged, thanks! 
👍

File: pages/common/asar.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# asar
+
+> A file archiver for the Electron platform.
```
Comment (sbrl) [inline pages/common/asar.md]: I know this got merged already, but what's the `Electron platform` in this instance, and how does that relate to file archiving? The only _Electron_ I'm aware of is the shell that the _Atom_ editor uses.

I'm confused.

File: pages/common/asar.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Extract an archive:
+
+`asar extract {{archived.asar}}`
+
+- Extract a specific file from an archive:
+
+`asar extract-file {{archived.asar}} {{file}}`
+
+- List files of an archive:
```
Comment (sbrl) [inline pages/common/asar.md]: Perhaps ...`inside an archive:` would read better?

File: pages/common/asar.md (hunk 1 above)
Comment (sbrl) [inline pages/common/asar.md]: I know this got merged already, but what's the `Electron platform` in this instance, and how does that relate to file archiving? The only _Electron_ I'm aware of is the shell that the _Atom_ editor uses.

I'm confused.

Comment (idleberg) [reply pages/common/asar.md]: It's the same thing and its standard archive format.

https://github.com/electron/asar

I don't know if it's worth mentioning that's made for Electron. To me it seems more useful than the official description „Simple extensive tar-like archive format with indexing”, but that's up to you guys :)

File: pages/common/asar.md (hunk 2 above)
Comment (idleberg) [reply pages/common/asar.md]: I agree. Or perhaps streamline it with other archivers, e.g. `7z` which uses „List the contents of an archive file”

File: pages/common/asar.md (hunk 1 above)
Comment (sbrl) [reply pages/common/asar.md]: Opened #2325.

This description does seem more useful, I'm just curious as to whether we can clarify it a bit more :-)

**Pull request merged**